docs/StardustDocs/topics/gettingStartedGradleAdvanced.md

```diff
@@ -168,6 +171,16 @@ dependencies {
 
 </tabs>
 
+<warning>
```

Comment:
I remember recommendation to not use warnings in similar cases

Reply:
which tags are available to use instead?

Reply:
I think plain text is good

Reply:
There's also `note`: https://www.jetbrains.com.cn/en-us/help/writerside/semantic-markup-reference.html#note